crates/bevy_winit/src/lib.rs

```diff
@@ -492,11 +489,6 @@ fn handle_winit_event(
                             event_loop.set_control_flow(ControlFlow::Wait);
                         }
                     }
-
-                    // Trigger the next redraw to refresh the screen immediately if waiting
-                    if let ControlFlow::Wait = event_loop.control_flow() {
-                        runner_state.redraw_requested = true;
-                    }
```

Comment:
why was this removed? it seems unrelated

Reply:
Accidentally removed it when I was dealing with merge conflicts. It's been put back.